### PR TITLE
v8.1.1: Turn off hash agg spill by default

### PR DESCRIPTION
**Pull request by xzhangxian1008**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [ ] master (the latest development version)
- [ ] v8.3 (TiDB 8.3 versions)
- [ ] v8.2 (TiDB 8.2 versions)
- [ ] v8.1 (TiDB 8.1 versions)
- [ ] v8.0 (TiDB 8.0 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: system-variables.md [hunk 1]
```diff
@@ -2185,8 +2185,8 @@ mysql> SELECT job_info FROM mysql.analyze_jobs ORDER BY end_time DESC LIMIT 1;
 - 是否持久化到集群：是
 - 是否受 Hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value) 控制：否
 - 类型：布尔型
-- 默认值：`ON`
-- 这个变量用来控制 TiDB 是否支持并行 HashAgg 进行落盘。当该变量设置为 `ON` 时，并行 HashAgg 将支持落盘。该变量将在功能正式发布后废弃。
+- 默认值：`OFF`
+- 这个变量用来控制 TiDB 是否支持并行 HashAgg 进行落盘。当该变量设置为 `ON` 时，并行 HashAgg 将支持落盘。默认值在 v8.1.0 为`ON`，从 v8.1.1 起为`OFF`。该变量将在功能正式发布后废弃。
```
Comment (qiancai) [inline system-variables.md:2189]:
```suggestion
- 默认值：在 v8.1.0 为 `ON`，从 v8.1.1 起为`OFF`
- 这个变量用来控制 TiDB 是否支持并行 HashAgg 进行落盘。当该变量设置为 `ON` 时，并行 HashAgg 将支持落盘。该变量将在功能正式发布后废弃。
```

File: system-variables.md [hunk 2]
```diff
@@ -2185,7 +2185,7 @@ mysql> SELECT job_info FROM mysql.analyze_jobs ORDER BY end_time DESC LIMIT 1;
 - 是否持久化到集群：是
 - 是否受 Hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value) 控制：否
 - 类型：布尔型
-- 默认值：`ON`
+- 默认值：在 v8.1.0 为 `ON`，从 v8.1.1 起为 `OFF`
```
Comment (qiancai) [inline system-variables.md:2188]:
```suggestion
- 默认值：在 v8.1.0 为 `ON`，v8.1.1 以及之后的 8.1 修订版本为 `OFF`
```

**Review by XuHuaiyu — APPROVED** [review]

**Review by qiancai — APPROVED** [review]

File: system-variables.md [hunk 3]
```diff
@@ -2185,7 +2185,7 @@ mysql> SELECT job_info FROM mysql.analyze_jobs ORDER BY end_time DESC LIMIT 1;
 - 是否持久化到集群：是
 - 是否受 Hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value) 控制：否
 - 类型：布尔型
-- 默认值：`ON`
+- 默认值：在 v8.1.0 中为 `ON`，在 v8.1.1 及后续的 8.1 修订版本中为 `OFF`
```
Comment (lilin90) [inline system-variables.md:2188]: [TiDB 版本规则 | PingCAP 文档中心](https://docs.pingcap.com/zh/tidb/stable/versioning#%E9%95%BF%E6%9C%9F%E6%94%AF%E6%8C%81%E7%89%88%E6%9C%AC) 里对 patch 的叫法为“补丁版本”，建议保持一致。
```suggestion
- 默认值：在 v8.1.0 中为 `ON`，在 v8.1.1 及后续的 8.1 补丁版本中为 `OFF`
```

**Pull request merged**